pulsar-client/src/main/java/org/apache/pulsar/client/api/Authentication.java

```diff
@@ -43,9 +43,7 @@
      * Configure the authentication plugins with the supplied parameters
      *
      * @param authParams
-     * @deprecated This method will be deleted on versin 2.0, instead please use configure(String encodedAuthParamString) which is in EncodedAuthenticationParameterSupport for now and will be integrated into this interface.
      */
-    @Deprecated
```

Comment:
Don't we deprecate this method?

Reply:
The problem is that we don't have a replacement yet for it. If you use `AuthenticationTLS` there's no way to avoid the deprecated method.

Reply:
OK. I understood.